packages/mobile/src/components/collection-list/CollectionList.tsx

```diff
@@ -11,6 +11,8 @@ import { AddCollectionCard } from './AddCollectionCard'
 import { CollectionCard } from './CollectionCard'
 import { CollectionCardSkeleton } from './CollectionCardSkeleton'
 
+const MemoizedCollectionCard = memo(CollectionCard)
```

Comment:
oh heck yeah! I always forget this is a thing

Reply:
will we run into trouble with stale cover photos and stats?

Reply:
no because the cards themselves will respond to redux changes, they just dont rerender due to renderItem being called